src/test/validator.test.ts

```diff
@@ -0,0 +1,10 @@
+import { FormValidator } from '../validator';
+
+test('return true if passwords match', () => {
```

Comment:
poczytaj o parametrized tests jest. https://medium.com/@Klaymant/parameterized-tests-with-typescript-jest-19f45a4a2925
Poszukam jeszcze czegos lepszego ale jak nie to to moze byc.